wolfhsm/wh_packet.h

```diff
@@ -507,17 +534,20 @@ typedef struct WOLFHSM_PACK wh_Packet_she_verify_mac_req
 typedef struct WOLFHSM_PACK wh_Packet_she_verify_mac_res
 {
     uint8_t status;
+    uint8_t padding[3];
 } wh_Packet_she_verify_mac_res;
 #endif
 
+
 /* use packed structs so we can read a packet in directly */
-typedef struct WOLFHSM_PACK whPacket
+typedef struct whPacket
 {
     /* header */
     int32_t rc;
     uint16_t flags;
+    uint16_t spare;
```

Comment:
lets have consistent names for padding bytes. Something unique and `grep`-able. How about `WHPADBYTE` or `whPadByte` or something like that